src/lib/libaddfunction.js

```diff
@@ -151,15 +151,17 @@ addToLibrary({
     }
     // Grow the table
     try {
+      var growBy = {{{ from64Expr('wasmTable.length') }}};
+      freeTableIndexes.push(...Array.from({ length: growBy }, (_, i) => {{{ from64Expr('wasmTable.length') }}} + i));
```

Comment:
It would use less memory to record the last used entry separately. Also I'm not sure we want to grow the table exponentially, @sbc100 was suggesting linear or quadratic.

Reply:
Okay, I have changed the behavior. Unfortunately, the last commit tests failed, and I did not get a clue what was wrong. (It must be something with the templating.) Maybe the new one is better.
Regarding linear or quadratic scaling, is the initial size of the table somewhere available? We need it to achieve linear or quadratic scaling. I also think it is important that the growth is in a similar order of magnitude as the current size.